.circleci/config.yml

```diff
@@ -29,31 +34,57 @@ jobs:
             rm google-chrome.deb
 
       - run:
-          name: Docker login
-          command: echo ${DOCKER_REGISTRY_PASSWORD} | docker login -u ${DOCKER_REGISTRY_USERNAME} --password-stdin ghcr.io
+          name: Update docker and docker compose
+          command: |
+            sudo apt-get purge docker-ce containerd.io
+            curl -fsSL https://get.docker.com -o get-docker.sh
+            sudo sh get-docker.sh
+            sudo curl -L "https://github.com/docker/compose/releases/download/1.26.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+            sudo chmod +x /usr/local/bin/docker-compose
 
       - run:
-          name: Spin container via docker compose
-          command: docker-compose up -d
+          name: checkout scalar-samples repository as it contains the script to spin up scalardl environment and easier for us to maintain the CI
+          command: |
+            git init
+            git pull https://${GHCR_USERNAME}:${GHCR_PAT}@github.com/scalar-labs/scalar-samples.git
```

Comment:
@feeblefakie @jnmt 
We realized that we could use the ${GHCR_PAT} to authenticate instead of using the ${GITHUB_PASSWORD}. 
If you are ok with this change, we can do the same modification as well on the Node sdk side.